tools/make-v8.sh

```diff
@@ -1,38 +1,53 @@
 #!/bin/bash
 
-
-git_origin=$(git config --get remote.origin.url | sed 's/.\+[\/:]\([^\/]\+\/[^\/]\+\)$/\1/')
-git_branch=$(git rev-parse --abbrev-ref HEAD)
-v8ver=${1:-v8} #default v8
-svn_prefix=https://github.com
-svn_path="$svn_prefix/$git_origin/branches/$git_branch/deps/$v8ver"
-#svn_path="$git_origin/branches/$git_branch/deps/$v8ver"
-gclient_string="solutions = [{'name': 'v8', 'url': '$svn_path', 'managed': False}]"
+if [ $# -eq 0 ]; then
+  # Default branch is master
+  BRANCH="master"
+else
+  # eg: 5.4-lkgr
+  BRANCH="$1"
+fi
```

Comment:
We may need a bit more doc with respect to this. I interpret this to be the v8 branch from which we are going to get the additional tools required to build.  So while we may be testing Node.js in branch X, in this case the branch refers to the v8 branch to get the tools from.  If my interpretation is correct, adding some explanation of that would be useful.